setup.py

```diff
@@ -41,15 +37,17 @@ def read(*rnames):
     version=version,
     python_requires='>=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*,!=3.5.*',
     packages=find_packages(exclude=['ez_setup', 'examples', 'tests']),
-    package_data=dict(rst2pdf=['styles/*.json',
-	'styles/*.style',
-	'images/*png',
-	'images/*jpg',
-	'templates/*tmpl'
-	]),
+    package_data={
+        "rst2pdf": [
+            "styles/*.json",
+            "styles/*.style",
+            "images/*png",
+            "images/*jpg",
+            "templates/*tmpl",
+        ]
+    },
     include_package_data=True,
-    dependency_links=[
-    ],
+    dependency_links=[],
     install_requires=install_requires,
     tests_require=tests_require,
     extras_require=dict(
```

Comment:
If this PR is the final step for Python 3 compatibility (or at least close enough to call it officially supported), I would also suggest that you add the [trove classifiers for Python 3](https://pypi.org/pypi?%3Aaction=list_classifiers):

```
Programming Language :: Python :: 3
Programming Language :: Python :: 3.5
Programming Language :: Python :: 3.6
Programming Language :: Python :: 3.7
```